From ddbfe0a98144945fc0e557ce88b94f1e3ee55d6e Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 31 Mar 2018 12:59:53 +0100
Subject: [PATCH 33/38] Use find_program to locate ant

This is necessary for Xcode builds since Xcode scrubs PATH
---
 src/gwt/CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/gwt/CMakeLists.txt b/src/gwt/CMakeLists.txt
index f969ebac92..77db07cde7 100644
--- a/src/gwt/CMakeLists.txt
+++ b/src/gwt/CMakeLists.txt
@@ -36,11 +36,15 @@ if(DEFINED ENV{GWT_BUILD})
 endif()
 
 if(GWT_BUILD)
+   find_program(ANT ant)
+   if(NOT ANT)
+      message(FATAL_ERROR "ant not found!")
+   endif()
    add_custom_target(gwt_build ALL)
    add_custom_command(
       TARGET gwt_build
       WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
-      COMMAND ant -Dbuild.dir="${GWT_BIN_DIR}"
+      COMMAND ${ANT} -Dbuild.dir="${GWT_BIN_DIR}"
                   -Dwww.dir="${GWT_WWW_DIR}"
                   -Dextras.dir="${GWT_EXTRAS_DIR}"
                   -Dlib.dir="${GWT_LIB_DIR}")
-- 
2.23.0

